src/frequenz/sdk/timeseries/logical_meter/logical_meter.py

```diff
@@ -0,0 +1,111 @@
+"""A logical meter for calculating high level metrics for a microgrid.
+
+Copyright
+Copyright © 2022 Frequenz Energy-as-a-Service GmbH
+
+License
+MIT
+"""
+
+import asyncio
+import uuid
+from typing import Dict, List
+
+from frequenz.channels import Broadcast, Receiver, Sender
+
+from ...actor import ChannelRegistry, ComponentMetricRequest
+from ...microgrid.component import ComponentMetricId
+from ...timeseries import Sample
+from ._formula_engine import FormulaBuilder, FormulaEngine
+
+
+class LogicalMeter:
+    """A logical meter for calculating high level metrics for a microgrid.
```

Comment:
... in a microgrid
would that sound better?

Reply:
updated.